src/yb/tools/yb-admin_cli.cc

```diff
@@ -83,6 +83,21 @@ CHECKED_STATUS ChangeBlacklist(ClusterAdminClientClass* client,
   return Status::OK();
 }
 
+CHECKED_STATUS MasterLeaderStepDown(
+    ClusterAdminClientClass* client,
+    const ClusterAdminCli::CLIArguments& args) {
+  if (args.size() < 3) {
+    return ClusterAdminCli::kInvalidArguments;
+  }
+  string leader_uuid;
+  RETURN_NOT_OK_PREPEND(client->GetMasterLeaderInfo(&leader_uuid), "Could not locate master leader!");
+  if (leader_uuid.empty()) {
+    return STATUS(ConfigurationError, "Could not locate master leader!");
+  }
+  return client->MasterLeaderStepDown(
```

Comment:
`leader_uuid` should be used as first argument for this function call instead of empty string.
Also please fix codding style. i.e. write function call with arguments on single line as it is less than 100 symbols